packages/policy/src/changer.ts

```diff
@@ -77,6 +78,7 @@ export async function addCodeOfConduct(
     upstreamOwner: owner,
     upstreamRepo: repo,
     fork: false,
+    branch: `policy-bot-${uuid()}`,
```

Comment:
One thing that comes to mind, is we'll perhaps want a bot pretty fast that cleans up dead branches if we're using this as a pattern.

Reply:
Good news here is that merging the PR works the same as any other contribution.  The "delete branch after merge" flag is one we should broadly enable by default for all our repos I think.